Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/CapsuleManipulatorRequestBus.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/ComponentBus.h>
+
+namespace AZ
+{
+    class Quaternion;
+} // namespace AZ
+
+namespace AzToolsFramework
+{
+    //! Interface for handling capsule manipulator requests.
+    //! Note that radius requests are handled by RadiusManipulatorRequests.
+    class CapsuleManipulatorRequests: public AZ::EntityComponentBus
+    {
+    public:
+        //! Get the height of the capsule shape.
+        virtual float GetHeight() const = 0;
+        //! Get the rotation of the capsule shape relative to the manipulator space.
+        virtual AZ::Quaternion GetRotationOffset() const = 0;
```

Comment:
It's not needed, because the capsule manipulators can only change the height, radius and translation offset (they can't change the rotation). But you do need to know the rotation offset in order to correctly update the translation offset.